libraries/skalibs/skalibs.info

```diff
@@ -1,8 +1,8 @@
 PRGNAM="skalibs"
-VERSION="2.13.1.1"
+VERSION="2.14.0.0"
 HOMEPAGE="https://skarnet.org/software/skalibs/"
-DOWNLOAD="https://skarnet.org/software/skalibs/skalibs-2.13.1.1.tar.gz"
-MD5SUM="2ede218c3583777cefcfc50a82683fbd"
+DOWNLOAD="https://skarnet.org/software/skalibs/skalibs-2.14.0.0.tar.gz"
+MD5SUM="be8d8069ec9162b41b95a0907ea8c2d5"
```

Comment:
@mumahendras3 Seems this md5sum is now:

`34e15729afc9428cf1814ffebb99543a  skalibs-2.14.0.0.tar.gz` Could you confirm and update please?